samza-sql/src/main/java/org/apache/samza/sql/translator/ScanTranslator.java

```diff
@@ -92,6 +93,15 @@ void translate(final TableScan tableScan, final TranslatorContext context,
     final String streamName = sqlIOConfig.getStreamName();
     final String source = sqlIOConfig.getSource();
 
+    final boolean isRemoteTable = sqlIOConfig.getTableDescriptor().isPresent() &&
+        (sqlIOConfig.getTableDescriptor().get() instanceof RemoteTableDescriptor);
+
+    // We don't need to define input stream descriptor for a remote table as the table descriptor is already defined by
```

Comment:
This is little bit confusing on why only for RemoteTable, ScanTranslator doesn't do anything. It should be no-op for both Remote and local table right?